components/calendar/calendar.js

```diff
@@ -408,6 +445,19 @@ class Calendar extends LocalizeCoreElement(RtlMixin(LitElement)) {
 		getCalendarData();
 	}
 
+	get dayInfos() {
```

Comment:
I figured out what was happening... it was because the update to `dayInfos` was happening inside of the event handler that was dispatched from within `willUpdate`, so an update was already in progress and so Lit would not trigger another one. Adding a `setTimeout` to the assignment in the event handler revealed what was happening. But the internal Map is removed now, so this is moot now.